Console/Command/NodesValidatorCommand.php

```diff
@@ -140,7 +146,10 @@ private function getInvalidNodes(array $nodes): array
 
         /** @var NodeInterface $node */
         foreach ($nodes as $node) {
-            $this->validator->validate([$node->getNodeId() => $node->getData()]);
+            $this->validator->validate(
+                [$node->getNodeId() => $node->getData()],
+                $this->treeTrace
```

Comment:
Not sure if passing this is required, can you double check?

Reply:
it seems to be not needed, thanks!